pkgs/build-support/fetchzip/default.nix

```diff
@@ -5,7 +5,7 @@
 # (e.g. due to minor changes in the compression algorithm, or changes
 # in timestamps).
 
-{ lib, fetchurl, unzip, glibcLocalesUtf8 }:
+{ lib, fetchurl, unpackZip ? true, unzip, glibcLocalesUtf8 }:
```

Comment:
Usually parameters enabling optional features are called [`withFoo`](https://github.com/NixOS/nixpkgs/blob/98426e81f4e2b8446c9ccb1fca584adb2c6a804b/pkgs/os-specific/linux/systemd/default.nix#L88), so I'd have expected this parameter to be named `withUnzip` instead of `unpackZip`.

Reply:
Renamed.